docs/chains/algorand.md

```diff
@@ -0,0 +1,22 @@
+# `algorand`
+
+`algorand` is a [Chain](/chains) connector in Hummingbot Gateway that helps users connect to DEXs on various networks on the Algorand blockchain.
+
+## 📁 Connector Info
+
+* Folder: [/gateway/src/chains/algorand](https://github.com/hummingbot/gateway/tree/main/src/chains/algorand)
+* Configs: [/gateway/src/templates/algorand.yml](https://github.com/hummingbot/gateway/tree/main/src/templates/algorand.yml)
+* Maintainer: [CoinAlpha](https://coinalpha.com)
```

Comment:
check with Loragene in teh #coinalpha channel on Discord if Algorand is paying CoinAlpha for maintenance. If not, CoinAlpha should not be here